plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/StandardSQLDialectQueryGenerator.java

```diff
@@ -103,6 +106,38 @@ public void appendConditionString(
         appendConditionString(filter, constraints, dataSource, conditionTable, query, inlineCriteria, subQuery);
     }
 
+    @Override
+    public String addFiltersToQuery(
+        @Nullable DBRProgressMonitor monitor,
+        final @NotNull DBPDataSource dataSource,
```

Comment:
IMHO this looks better
```suggestion
        @NotNull final DBPDataSource dataSource,
```

Reply:
It's not required at all TBH